documentation/apptransportsecurity/index.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: documentation
+id: documentation
+title: Sparkle App Transport Security
+---
+## App Transport Security
+
+Apple has deprecated insecure HTTP in OS X 10.11 (El Capitan). Applications compiled with 10.11 SDK are by default **blocked from using HTTP** and will not be able to download any updates over HTTP. You **must** either:
+
+* Switch to HTTPS entirely. Change the appcast URL in Info.plist and download URLs in the appcast to use HTTPS. This is recommended, and you can get free certificates from [Let's encrypt](https://letsencrypt.org/).
+
+* Add an exception to Info.plist in your app if you can't stop using HTTP yet. [The exception properties are in Apple's documentation](https://developer.apple.com/library/prerelease/ios/technotes/App-Transport-Security-Technote/).
```

Comment:
Replace 'ios' with 'mac' here. While the page is the same... this is an OS X project. ;)